### ISSUE TITLE
Linux kernel panic while booting for the first time, on a Dell T340 Server with UEFI boot

### ISSUE THREAD

**Issue by howdy-world** [pocopico/tinycore-redpill#129]
Hi, i'm having a Dell T340 server. I used the UEFI version of the image. After it passes grub, the screen freezes and later on reboots, and so on.

I tried enabling serial and increase log level, by editing the command line option inside grub, to add `loglevel=7 console=ttyS0,115200n8`, then i see the following boot logs. There's a kernel panic shortly after it booted up. 

See attached file for the boot log i captured, using the serial-over-LAN/SSH feature of the Dell iDRAC:
[kmsg.log](https://github.com/pocopico/tinycore-redpill/files/9275985/kmsg.log)

The server has  Xeon E-2224, 8GB x2 UDIMM ECC, booting with only one USB drive, and one 100GB SATA SSD attached to the motherboard (which i assumed coming out of south bridge)

Any idea how to troubleshoot this issue? Thanks!

**Comment by PeterSuh-Q3** [comment]
This problem is not a kernel panic in pocopico's TCRP boot process.
Tinycore Linux kernel seems to panic when booting.
It seems right to ask on the Tinycore Linux forum below.
http://forum.tinycorelinux.net/

**Comment by howdy-world** [comment]
@PeterSuh-Q3 , you're right. I am able to reproduce the crash using simply the official TinyCore 12.0 iso. (Furthermore, the TinyCore 13.0 iso has the same crash for me)

Looks like it's time to submit a report to TC

**Comment by pocopico** [comment]
You can still try to create your loader on your desktop and then run this on your server
